src/fmu/dataio/datastructure/meta/meta.py

```diff
@@ -81,41 +112,59 @@ def _check_for_non_ascii_in_path(cls, values: Dict) -> Dict:
 
 
 class Parameters(RootModel):
+    """
+    The ``parameters`` block contains the parameters used in a realization. It is a
+    direct pass of ``parameters.txt`` and will contain key:value pairs representing the
+    design parameters.
```

Comment:
I would leave out the word `design` here since parameters.txt is also created for AHM runs, not only for sensitivity runs.